tools/prlint/module.ts

```diff
@@ -0,0 +1,37 @@
+import * as fs from 'fs-extra';
+import * as glob from 'glob';
+import * as path from 'path';
+
+const modules: string[] = [];
+
+export function findModulePath(fuzz: string): string {
+  if (modules.length === 0) {
+    const repoRoot = path.join(__dirname, '..', '..');
```

Comment:
To avoid this breaking if/when we change the repo structure we shouldn't  use relative references, better to implement `walk`  to find `lerna.json` file, similar to what we did [here](https://github.com/aws/aws-cdk/blob/master/tools/ubergen/bin/ubergen.ts#L76).

Reply:
Im going to tag @rix0rrr who recently tried to eliminate all relative references in our code to support his new build tool sh he can weigh in.

Reply:
> can we add an argument to validatePr (and threaded to here) that would be populated by `github.context.workspace` when run as an action?

If I understand Github actions correctly, the context will need to be passed in as an argument to the `prlint` github action and then threaded through. 
Is that correct, or is there something simpler I'm missing where the context can be pulled out of a global map?

If so, I'll reiterate - what's the RoI for something that may never have to change? Or change once.

> eliminate all relative references in our code to support his new build tool

Sure! This package is not in the dep tree or any of our CDK modules, so shouldn't matter?

Reply:
Ah, yes I think I misunderstood where the context would be used. The [GITHUB_WORKSPACE environment variable](https://docs.github.com/en/actions/reference/environment-variables#default-environment-variables) would actually be available in the runner.

At this point, we've probably spent more time discussing this change than what it would take to implement so I'm fine with keeping as is :) (pending Rico's comment)

Reply:
Thanks for thinking of me Neta! 

Yes, I think this change will not be breaking. It won't execute during the build, so it won't affect the build.

Nevertheless, I'm not particularly fond of it either. This is exactly the style of code that made the build annoying to refactor in the first place. Saying: "the repository's root can be found *relative to my own source files*" will preclude this package from being moved around, shared between repositories, vended to NPM, etc.

For no real benefit, either. You could be making assumptions on `process.cwd()` instead (don't depend on where the program comes from, depend on where it's being run!), and if you feel scared you can always go for  `findUp()`-style solution. Has the same result, but feels more robust, ya know?

Reply:
> This is exactly the style of code that made the build annoying to refactor in the first place. Saying: "the repository's root can be found relative to my own source files" will preclude this package from being moved around, shared between repositories, vended to NPM, etc.

> `findUp()`-style solution. Has the same result, but feels more robust, ya know?

See my original response on this thread about why I think these arguments don't bear out in terms of cost-benefit.

Nevertheless, since all of you feel strongly about this, I'll change it so it's not reliant on relative paths.